app/config.py

```diff
@@ -686,3 +687,7 @@ def read_partner_dict(var: str) -> dict[int, str]:
 ALIAS_TRASH_DAYS = int(os.environ.get("ALIAS_TRASH_DAYS", 30))
 ALLOWED_OAUTH_SCHEMES = get_env_csv("ALLOWED_OAUTH_SCHEMES", "auth.simplelogin,https")
 MAX_EMAIL_FORWARD_RECIPIENTS = int(os.environ.get("MAX_EMAIL_FORWARD_RECIPIENTS", 30))
+
+MASTER_ENC_KEY = bytes.fromhex(os.environ.get("MASTER_ENC_KEY_HEX"))
+MAC_KEY = bytes.fromhex(os.environ.get("MAC_KEY_HEX"))
+ABUSER_HKDF_SALT = base64.b64decode(os.environ.get("ABUSER_HKDF_SALT"))
```

Comment:
These three can be set to nothing for people that self host. If they aren't set to anything we should a default value derived from the flask secret for instance. This way people self hosting won't have their installations failing.

We could do something like 
```python
def decode_hex_data(key: string, default: bytes) -> bytes:
  data = os.environ.get(key)
  if not data:  
   return default
  else;
    return bytes.fromhex(data)

MASTER_ENC_KEY = decode_hex_data("MASTER_ENC_KEY_HEX", (FLASK_SECRET + 'enckey').encode("UTF-8+")
MAC_KEY = decode_hex_data("MAC_KEY", (FLASK_SECRET + 'mackey').encode("UTF-8+")
ABUSER_HKDF_SALT = decode_hex_data("ABUSER_HKDF_SALT", (FLASK_SECRET + 'absalt').encode("UTF-8+")
```